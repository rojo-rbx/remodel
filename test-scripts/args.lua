for i = 1, select("#", ...) do
	print("Arg", i, (select(i, ...)))
end