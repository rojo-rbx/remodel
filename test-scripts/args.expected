Arg	1	arg1
Arg	2	arg2
Arg	3	arg3